recipe/meta.yaml

```diff
@@ -16,10 +16,10 @@ source:
     - patches/0006-popcnt_fix.patch                     # [blas_impl == "mkl" and win]
 
 build:
-  number: 1
+  number: 2
   # numpy 1.20.0 no longer supports Python 3.6: https://numpy.org/doc/stable/release/1.20.0-notes.html
   # "The Python versions supported for this release are 3.7-3.9, support for Python 3.6 has been dropped"
-  skip: True  # [(blas_impl == 'openblas' and win) or py2k or py<37]
+  skip: True  # [(blas_impl == 'openblas' and win) or py<37 or py>311]
```

Comment:
why we want to skip python >3.11 (not that we even support 3.11 at all)?  Maybe add cause to comment

Reply:
numpy 1.21.x set Python upper bound <3.11, see https://github.com/numpy/numpy/commit/1e8d6a83985f3191c63963414981743adc4353cf
I've added a comment to the recipe